automation_tools/__init__.py

```diff
@@ -2236,11 +2236,7 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         puppet4_repo = os.environ.get('PUPPET4_REPO')
         if puppet4_repo:
             execute(create_custom_repos, puppet4_repo=puppet4_repo, host=host)
-    # Sat6.4 and above: enable internal foreman-maintain repo for installation
-    if sat_version != '6.3' and not distribution.endswith('cdn'):
-        maintain_repo = os.environ.get('MAINTAIN_REPO')
-        if maintain_repo:
-            execute(create_custom_repos, maintain_repo=maintain_repo, host=host)
```

Comment:
what if distribution is internal/ internal repo profile? how maintain repo will be added?

Reply:
for INTERNAL REPOFILE, the repo file we get has an entry for the maintenance repo. 

For INTERNAL, I probably could add this code back in, in the `downstream_install` method

Reply:
@ntkathole fixed this by moving it to `downstream_install` method and adding the options for a MAINTAIN_BASE_URL to installer job see also https://github.com/SatelliteQE/robottelo-ci/pull/1570

Reply:
@pgagne what about not including in downstream_install and let it be where it is now (in product_install)
and you would just change if clause to: 
```
if sat_version != '6.3' and distribution.endswith('downstream'):
```
I don like introducing new env var to downstream_install() 
I have a plan :-) to unify all ```*_install() ``` into one or two common ways of install

Reply:
Since this PR only attempts to narrow the initial condition that is way too broad:
```
if sat_version != '6.3' and not distribution.endswith('cdn'):
```
to 
```
if sat_version != '6.3' and distribution.endswith('downstream'):
```
Why such a rewrite if you only want to narrow down one simple condition ? ;-)